apps/main/src/entities/chain/model/chain-validation.ts

```diff
@@ -1,5 +1,5 @@
 import type { ChainQueryParams } from '@/entities/chain/types'
-import { createValidationSuite } from '@/shared/validation'
+import { createValidationSuite } from '@/shared/curve-lib'
```

Comment:
```suggestion
import { createValidationSuite } from '@/shared/curve-lib/validation'
```
Once you fix the exports in curve-lib this and similar imports should look like this to comply with FSD.